src/Widgets/Shapes/ShapeRectangle.vala

```diff
@@ -35,11 +35,22 @@ public class GtkCanvas.ShapeRectangle : GtkCanvas.CanvasItem {
             if (value == "") return;
 
             _color = value;
-            clutter_color = Clutter.Color.from_string (value);
+            _clutter_color = Clutter.Color.from_string (value);
```

Comment:
and also set the property instead of the field (`_`), so that you get the nulchecks and all